whelktool/scripts/cleanups/2019/02/move-holding-Jox-to-Jo/move-jox-holdings.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * This moves all holdings for Jox to Jo.
+ *
+ * See LXL-2248 for more info.
+ *
+ */
+String CURRENT_SIGEL = 'https://libris.kb.se/library/Jox'
+String NEW_SIGEL = 'https://libris.kb.se/library/Jo'
+
+PrintWriter scheduledForMove = getReportWriter("scheduled-for-move")
+PrintWriter notMoved = getReportWriter("refused-to-move")
+
+selectBySqlWhere("""
+        collection = 'hold' AND
+        data#>>'{@graph,1,heldBy,@id}' = '${CURRENT_SIGEL}'
+    """, silent: false) { hold ->
+    def (record, thing, work) = hold.graph
+    boolean wouldCreateDupe = false
+
+    selectBySqlWhere("""
+        collection = 'hold' AND
+        data#>>'{@graph,1,itemOf,@id}' = '${thing["itemOf"][ID]}' AND
+        data#>>'{@graph,1,heldBy,@id}' = '$NEW_SIGEL'
+    """, silent: true) { other_hold ->
+        wouldCreateDupe = true
+    }
+
+    if (wouldCreateDupe) {
```

Comment:
Do we really want to keep these Jox-holdings?

Reply:
Yup. See comment in ticket.

Reply:
All good then! :white_check_mark: